cmd/init.js

```diff
@@ -1,16 +1,35 @@
-const fsExstra = require('fs-extra');
+const fsExtra = require('fs-extra');
```

Comment:
You are using 2 separate libraries for file operations. Chose one and remove the other. I recommend you to stick to `fs`.

Reply:
NodeJs  fs lib does not provide option to copy a whole folder ( atleast I did not find ). fsExtra does and this is the reason  it was used by @varasu
The other option is to write our own custom method that does this using only the standard library. I am against this because it means more code to maintain and test.